apps/frontend/src/pages/Homepage.tsx

```diff
@@ -170,6 +181,57 @@ const Homepage = () => {
 
   return (
     <>
+      {charityEvent != "" && (
+        <div style={{ position: "relative", height: "80px" }}>
+          <Typography
+            variant="h5"
+            style={{
+              position: "relative",
+              zIndex: 4,
+              fontWeight: "bold",
+              paddingLeft: "14px",
+              height: "23px",
+            }}
+          >
+            Charity
+          </Typography>
+          <Typography
+            variant="h5"
+            style={{
+              position: "relative",
+              zIndex: 4,
+              fontWeight: "bold",
+              paddingLeft: "14px",
+            }}
+          >
+            Events:
+          </Typography>
+          <Typography
+            variant="h5"
+            style={{
+              position: "absolute",
+              zIndex: 4,
+              top: "23px",
+              left: "10%",
+            }}
+          >
+            {charityEvent}
+          </Typography>
```

Comment:
This will likely need to be changed so that it's a bit more responsive, I'll need to do this too so if you figure out a nice way to do it lmk!

_Rendered on iPhone 6_
![image](https://github.com/user-attachments/assets/9374fb0e-0324-4890-81bc-4190a5b2378a)

Reply:
Fixed